autofaiss/external/quantize.py

```diff
@@ -392,6 +396,11 @@ def build_partitioned_indexes(
         raise ValueError(f"nb_indices_to_keep must be > 0; Got {nb_splits_per_big_index}")
     if big_index_threshold < 1:
         raise ValueError(f"big_index_threshold must be > 0; Got {big_index_threshold}")
+    if index_path and not index_key:
```

Comment:
`if index_path is not None and not index_key:` instead?
Because index_path="" would pass the condition

Reply:
Good catch, yes :)